os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -131,6 +131,18 @@ void board_i2s_initialize(void)
 void board_spi_initialize(void)
 {
 #ifdef CONFIG_SPI
+
+#if defined(AMEBASMART_SPI0) && (AMEBASMART_SPI0 == 0)
+	struct spi_dev_s *spi;
```

Comment:
Your commit allows to enable both spi0 and spi1. Then, this local variable will be defined twice by spi0 and spi1.

Reply:
I have placed the struct at the CONFIG_SPI so it does not redefined